packages/salesforcedx-utils-vscode/src/workspaces/rootWorkspace.ts

```diff
@@ -9,6 +9,7 @@ import { join } from 'path';
 import { workspace, WorkspaceFolder } from 'vscode';
 import { SFDX_FOLDER } from '../types';
 
+// TODO: consolidate all of these into workspaceUtils
```

Comment:
Note there was already a file that has some of the same functionality as what's in workspaceUtils, but I didn't want to blow up this PR with the refactor.